resources/js/Pages/Praktikan.vue

```diff
@@ -2105,6 +2159,51 @@ export default {
           } 
         }
       });
+    },
+
+    formPassword: function($bool){
+      if($bool){
+        this.viewPassForm = true;
+      }else{
+        this.viewPassForm = false;
+        this.resetPass.password = '';
+        this.resetPass.repeatpass = '';
```

Comment:
```suggestion
      this.viewPassForm = $bool;
      if (!$bool) {
        this.resetPass.password = '';
        this.resetPass.repeatpass = '';
      }
```
You can optimize it this way actually

Reply:
thankyou bang for reminding this thing